src/mono/wasm/wasm.proj

```diff
@@ -1,6 +1,7 @@
-<Project DefaultTargets="Build">
-  <Import Project="Directory.Build.props" />
-  <Import Project="Directory.Build.targets" />
+<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="Build">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
Did you really need to add a tfm for this?

Reply:
I get:

NuGet.RestoreEx.targets(10,5): error : Invalid framework identifier ''. [/Users/vargaz/git/runtime/src/mono/wasm/wasm.proj]

otherwise.

Reply:
It seems like somehow it is being restored, interesting. @ViktorHofer any suggestion here?

Reply:
We're already doing the same thing in llvm.proj so this is fine.